### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by romani**

### CODE REVIEW

**Comment by strkkk** [comment]
@romani @rnveach please take a look. Crowdin added post processor for line wrapping. There a bit more changes, wrapping is applied to existing translations that were not wrapped, but it seems fine to me.
What do you think?

Please do not mind issues with escapes of single quotes, they should be resolved after https://github.com/checkstyle/checkstyle/pull/10957 is done (resync is needed)

File: src/main/resources/com/puppycrawl/tools/checkstyle/checks/annotation/messages_de.properties [hunk 1]
```diff
@@ -1,17 +1,25 @@
 annotation.incorrect.style=Die Annotation muss dem Stil ''{0}'' folgen.
-annotation.location=Die Annotation ''{0}'' hat eine unerwartete Einrückungstiefe von {1} (erwartet: {2}).
+annotation.location=Die Annotation ''{0}'' hat eine unerwartete Einrückungstiefe von {1} \
+                    (erwartet: {2}).
```
Comment (rnveach) [inline src/main/resources/com/puppycrawl/tools/checkstyle/checks/annotation/messages_de.properties:3]: Can we control indentation?

**Review by rnveach — COMMENTED** [review]
Too hard to see anything else with all these changes.

File: src/main/resources/com/puppycrawl/tools/checkstyle/checks/annotation/messages_de.properties (hunk 1 above)
Comment (strkkk) [reply src/main/resources/com/puppycrawl/tools/checkstyle/checks/annotation/messages_de.properties:3]: not sure, I can ask crowdin support about it. Is it a big issue?

**Comment by strkkk** [comment]
I closed PR and deleted branch to make sync again

**Pull request closed without merging**